content/en/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md

```diff
@@ -150,7 +150,7 @@ for these devices:
 ```gRPC
 // PodResources is a service provided by the kubelet that provides information about the
```

Comment:
Why wouldnt you change PodResources to PodResourcesLister on this line as well?

Reply:
Yep.
I forgot it.
I have fixed it.